cmd/stdlog_integration_test.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/cmd/tests"
+	"go.k6.io/k6/lib/testutils/httpmultibin"
+)
+
+// SetOutput sets the global log so it is racy with other tests
```

Comment:
Hmm maybe move it to a separate package then? After all, it currently might not be executed in parallel with the other tests, but it will still affect their `log` defaults :thinking: 

Though, now that I think about it, won't all e2e/integration tests have the same problem, since we always do that? :confused:

Reply:
>but it will still affect their log defaults

Sorry, I'm not getting it. What do you mean?

>Though, now that I think about it, won't all e2e/integration tests have the same problem, since we always do that? 

We have everything mocked in the GlobalState and we create it once per test, so we shouldn't have that kind of problem. Right?

Reply:
Yes, but when every test runs with that mocked `GlobalTestState`, it will still execute that line: https://github.com/grafana/k6/blob/367163be338f0d23b679585996aa2b13e3ec9cae/cmd/root.go#L81

Which will call stdlib's [`log.SetOutput`](https://pkg.go.dev/log#SetOutput) and potentially affect other tests... :disappointed: Not a lot we can do about it besides what you have already done here in making the test not parallel :+1: [It is thankfully synchronized](https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/log/log.go;drc=99f811ecca5674a6e6f423b97ea79f0a6eedf4b6;l=79), so at least that shouldn't be a big problem :thinking: 

It is just annoying that it is some global state we need to change for k6 to work correctly, and that we probably can't mock in any way :disappointed: Anyway, I will resolve this comment, your PR is handling this fine and other tests shouldn't be too affected by this :crossed_fingers: